code/datums/station_traits/neutral_traits.dm

```diff
@@ -496,24 +496,28 @@
 		shirt.forceMove(boxie)
 
 /// A box containing a skub, for easier carry because skub is a bulky item.
-/obj/item/storage/box/skub
-	name = "skub box"
-	desc = "A box to store your skub and pro-skub shirt in. A label on the back reads: \"Skubtide, Stationwide\"."
-	icon_state = "hugbox"
-	illustration = "skub"
-
-/obj/item/storage/box/skub/Initialize(mapload)
+/obj/item/storage/box/stickers/skub
```

Comment:
I'm not sure if a small sticker pack should contain the immensely big skub but w/e